packages/test-config/jest-dev.config.ts

```diff
@@ -14,6 +14,7 @@ const config: Config = {
         'jest-watch-typeahead/filename',
         'jest-watch-typeahead/testname',
     ],
+    workerThreads: true,
```

Comment:
For posterity: https://github.com/facebook/jest/issues/11617#issuecomment-1458155552